cookbook/persistance/key-value.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: "Storing key-value data on disk"
+permalink: /cookbook/persistence/key-value/
+---
+
+If you have a relatively small collection of key-values that you'd like to save, you can use the [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin.
+
+Normally we would have to write native platform integration for storing data for both platforms, but fortunately [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin can be used for that. Shared preferences plugin wraps `NSUserDefaults` on iOS and `SharedPreferences` on Android, providing a persistent store for simple data.
+
+## Setup
+
+Before you start, you need to add [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin to you `pubspec.yaml` file:
+
+```yaml
+dependencies:
+  flutter:
+    sdk: flutter
+  shared_preferences: "<newest version>"
+```
+
+## Saving data
+
+In order to access store simply use `SharedPreferences` class.
```

Comment:
"In order to access the store, we can use..."

In general, I'm trying to avoid the words "just" or "simply" -- I've gotten feedback from newcomers that even if something seems simple, it might be difficult to understand!